test/specs/rules/validate-indentation.js

```diff
@@ -156,6 +156,23 @@ describe('rules/validate-indentation', function() {
         ]);
     });
 
+    it('should not report errors for using tabs when validateIndentation is a number', function() {
+        checker.configure({ validateIndentation: 4 });
```

Comment:
So you want to use spaces for indentation but still using tabs? This would be quite unconventional.
![687474703a2f2f6576616465666c6f772e636f6d2f77702d636f6e74656e742f75706c6f6164732f323031312f30332f54616273537061636573426f74682e706e67](https://cloud.githubusercontent.com/assets/945528/12373689/3171f9d2-bc92-11e5-9269-46c79048da44.png)